keyboards/crkbd/keymaps/rarick/glcdfont.c

```diff
@@ -0,0 +1,233 @@
+  0x26, 0x49, 0x49, 0x49, 0x32, 0x00, 
+  0x03, 0x01, 0x7F, 0x01, 0x03, 0x00, 
+  0x3F, 0x40, 0x40, 0x40, 0x3F, 0x00, 
+  0x1F, 0x20, 0x40, 0x20, 0x1F, 0x00, 
+  0x3F, 0x40, 0x38, 0x40, 0x3F, 0x00, 
+  0x63, 0x14, 0x08, 0x14, 0x63, 0x00, 
+  0x03, 0x04, 0x78, 0x04, 0x03, 0x00, 
+  0x61, 0x59, 0x49, 0x4D, 0x43, 0x00, 
+  0x00, 0x7F, 0x41, 0x41, 0x41, 0x00, 
+  0x02, 0x04, 0x08, 0x10, 0x20, 0x00, 
+  0x00, 0x41, 0x41, 0x41, 0x7F, 0x00, 
+  0x04, 0x02, 0x01, 0x02, 0x04, 0x00, 
+  0x40, 0x40, 0x40, 0x40, 0x40, 0x00, 
+  0x00, 0x03, 0x07, 0x08, 0x00, 0x00, 
+  0x20, 0x54, 0x54, 0x78, 0x40, 0x00, 
+  0x7F, 0x28, 0x44, 0x44, 0x38, 0x00, 
+  0x38, 0x44, 0x44, 0x44, 0x28, 0x00, 
+  0x38, 0x44, 0x44, 0x28, 0x7F, 0x00, 
+  0x38, 0x54, 0x54, 0x54, 0x18, 0x00, 
+  0x00, 0x08, 0x7E, 0x09, 0x02, 0x00, 
+  0x18, 0x24, 0x24, 0x1C, 0x78, 0x00, 
+  0x7F, 0x08, 0x04, 0x04, 0x78, 0x00, 
+  0x00, 0x44, 0x7D, 0x40, 0x00, 0x00, 
+  0x20, 0x40, 0x40, 0x3D, 0x00, 0x00, 
+  0x7F, 0x10, 0x28, 0x44, 0x00, 0x00, 
+  0x00, 0x41, 0x7F, 0x40, 0x00, 0x00, 
+  0x7C, 0x04, 0x78, 0x04, 0x78, 0x00, 
+  0x7C, 0x08, 0x04, 0x04, 0x78, 0x00, 
+  0x38, 0x44, 0x44, 0x44, 0x38, 0x00, 
+  0x7C, 0x18, 0x24, 0x24, 0x18, 0x00, 
+  0x18, 0x24, 0x24, 0x18, 0x7C, 0x00, 
+  0x7C, 0x08, 0x04, 0x04, 0x08, 0x00, 
+  0x48, 0x54, 0x54, 0x54, 0x24, 0x00, 
+  0x04, 0x04, 0x3F, 0x44, 0x24, 0x00, 
+  0x3C, 0x40, 0x40, 0x20, 0x7C, 0x00, 
+  0x1C, 0x20, 0x40, 0x20, 0x1C, 0x00, 
+  0x3C, 0x40, 0x30, 0x40, 0x3C, 0x00, 
+  0x44, 0x28, 0x10, 0x28, 0x44, 0x00, 
+  0x4C, 0x90, 0x90, 0x90, 0x7C, 0x00, 
+  0x44, 0x64, 0x54, 0x4C, 0x44, 0x00, 
+  0x00, 0x08, 0x36, 0x41, 0x00, 0x00, 
+  0x00, 0x00, 0x77, 0x00, 0x00, 0x00, 
+  0x00, 0x41, 0x36, 0x08, 0x00, 0x00, 
+  0x02, 0x01, 0x02, 0x04, 0x02, 0x00, 
+  0x3C, 0x26, 0x23, 0x26, 0x3C, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x02, 0x06, 
+  0x06, 0x06, 0x06, 0x7F, 0xFF, 0xFF, 
+  0xFF, 0x07, 0x01, 0x01, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x80, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
+  0xE0, 0xF0, 0xF0, 0xF0, 0xE0, 0xEC, 
+  0xEE, 0xF7, 0xF3, 0x70, 0x20, 0x00, 
+  0x7C, 0x7C, 0x7C, 0x7E, 0x00, 0x7E, 
+  0x7E, 0x7E, 0x7F, 0x7F, 0x7F, 0x00, 
+  0x00, 0x80, 0xC0, 0xE0, 0x7E, 0x5B, 
+  0x4F, 0x5B, 0xFE, 0xC0, 0x00, 0x00, 
+  0xC0, 0x00, 0xDC, 0xD7, 0xDE, 0xDE, 
+  0xDE, 0xD7, 0xDC, 0x00, 0xC0, 0x00, 
+  0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 
```

Comment:
While I like the cowbow bebop reference...

Reply:
Thanks! Here's a bitmap of the font if you're interested. Looks great on the crkbd:

https://raw.githubusercontent.com/rarick/crkbd/master/fonts/cowboyBebop/cowboyBebop.bmp

Reply:
I usually use https://joric.github.io/qle/ for that sort of stuff. 

Generates and takes QMK OLED code.

Reply:
I don't think "see you space cowboy" is under any trademark/copyright, and I made everything there from scratch. I don't believe this could possibly be a legal issue, it's being distributed without profit. It's the legal equivalent of fanart.